.github/workflows/res-update-game.yml

```
@@ -201,61 +198,29 @@ jobs:
             ./tools/ResourceUpdater/libopencv_world4.4.11.0.dylib
             ./tools/ResourceUpdater/res_updater
 
-      - name: Download txwy
-        id: download-txwy
-        uses: actions/download-artifact@v6
-        continue-on-error: true
+      - name: Wait for resource cloning
+        uses: yogeshlonkar/wait-for-jobs@v0
         with:
-          name: txwy
-          path: ./tools/ResourceUpdater/Overseas/tw/gamedata/excel
+          ignore-skipped: 'false'
+          jobs: |
+            Download Official Resources
+            Download Overseas Resources
+            Download Taiwan Resources
```

```suggestion
            Download txwy
            Download Overseas
            Download Official
```

These names do not coincide with the `name:` of the download steps no?


From the README:

>     # Comma or newline separated list of names of job dependencies for this step, it must be `name:` property of job if set
>     # required: true
>     jobs: 'job1,job2,Job 3'


I'm not sure why the developer uses this format for the jobs (it seems to show they can be generic? So case insensitive and spaces can be omitted?

---

<https://github.com/MaaAssistantArknights/MaaAssistantArknights/blob/adb43c9736a7c73e99a12671975b0f8fb06263c4/.github/workflows/res-update-game.yml#L15>

<https://github.com/MaaAssistantArknights/MaaAssistantArknights/blob/adb43c9736a7c73e99a12671975b0f8fb06263c4/.github/workflows/res-update-game.yml#L47>

<https://github.com/MaaAssistantArknights/MaaAssistantArknights/blob/adb43c9736a7c73e99a12671975b0f8fb06263c4/.github/workflows/res-update-game.yml#L90>

---

> From the README:
> 
> > ```
> > # Comma or newline separated list of names of job dependencies for this step, it must be `name:` property of job if set
> > # required: true
> > jobs: 'job1,job2,Job 3'
> > ```
> 
> I'm not sure why the developer uses this format for the jobs (it seems to show they can be generic? So case insensitive and spaces can be omitted?

I read the source code, so this action use `name` to match the jobs (`job.name === name`):

<https://github.com/yogeshlonkar/wait-for-jobs/blob/708b4037b51e860d3d4dbb1f0ab098e9a9dbcd80/src/WaitForJobs.ts#L86-L94>

> ```typescript
>     private toCheck = (name: string, jobs: Job[]): Dependency => {
>         const js = jobs.filter(
>             job =>
>                 (this.asSuffix && job.name.endsWith(name)) ||
>                 (this.asPrefix && job.name.startsWith(name)) ||
>                 job.name === name
>         );
>         return new Dependency(name, js, this.ignoreSkipped);
>     };
> ```

---

And since they use `===`, then the `name` must be exactly the same.